qutebrowser/config/configtypes.py

```diff
@@ -844,12 +844,18 @@ class Directory(BaseType):
 
     typestr = 'directory'
 
+    def __init__(self, none_ok=False, special_values=None):
+        super().__init__(none_ok)
+        self._special_values = special_values or ''
```

Comment:
I think this should be a list rather than a string (this would also mean changing the configdata call above to `special_values=['%']`).

Also, could you maybe add a test for this? There's already `tests/config/test_configtypes.py:TestDirectory` so it should be easy to add that.